test/hzz2l2v/macro_Intr_MET_Syst/MakeSyst_custom_forMTandMET.C

```diff
@@ -137,7 +137,7 @@ void MakeSyst_custom(TFile* f_input, TFile* f_output, TString SystType, int proj
 	//The same axis than the one from limits: https://github.com/cms2l2v/2l2v_fwk/blob/master/bin/common/computeLimit.cc#L2587
 	Double_t eq0jets_axis_limits[] = {150, 225, 300, 375, 450, 525, 600, 725, 850, 975, 1100, 1350, 1600, 2100, 3000};
 	Double_t geq1jets_axis_limits[] = {150, 225, 300, 375, 450, 525, 600, 725, 850, 975, 1100, 1350, 1600, 2100, 3000};
-	Double_t vbf_axis_limits[] = {150, 225, 300, 375, 450, 525, 600, 725,  1100, 1350, 3000};
+	Double_t vbf_axis_limits[] = {150, 225, 300, 375, 450, 525, 600, 725,  1100, 3000};
```

Comment:
The new axis should be :
{150, 225, 300, 375, 450, 600, 750, 1100, 3000};
instead of:
{150, 225, 300, 375, 450, 525, 600, 725,  1100, 3000};
I.e, the bin 525 should be removed.

Thanks a lot for pushing all those changes Hugues.